_posts/2022-12-23-easily-verify-your-rust-in-ci-with-kani.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Easily verify your Rust in CI with Kani and Github Actions "
+---
+
+Formal verification tools, such as Kani, are valuable because they provide a way to definitively check, using mathematical techniques, whether a property of your code is true under all circumstances.  To put it colloquially: testing is great for catching the FIRST bug, formal verification is great for knowing you’ve caught the LAST bug.  
```

Comment:
Add a link to Kani when it's first mentioned.

Reply:
done